tech-debt.md

```diff
@@ -2,5 +2,88 @@
+* *Pull requests* are perfect for talking about tech debt in the context of a specific change. It is the PR owner's job to draw reviewers' attention to these considerations, and reviewers should ensure that PR owners are doing this correctly. PRs should outline:
+  * what existing technical debt the changes are impacted by
+  * what, if anything, the changes do to help address existing tech debt
+  * what technical debt the changes introduce or compound, and any justification for them
+  * what GitHub issues and/or Jiras have been raised to address any new debt in the future
+* *Guilds* can be used to see if people in other pods are having a similar experience with an existing piece of tech debt.
+
+### 4. Limit its influence
+
+Technical debt that is isolated has much less scope to interfere with other code. This tends to emerge naturally if code is loosely coupled and modular in nature.
+
+Say you have a module and its internal implementation is mess, but its interface is well thought out and does not change very often. This is an example of isolated technical debt. The messy internals will rarely interfere with work on the wider application. If however the interface becomes the subject of many changes, or it has a lot of bugs, it might be a good time to refactor or even just throw it out and start again.
+
+### 5. Be deliberate
+
+Whilst it is "okay" to take on technical debt, it has to be a deliberate decision. The short term gains of shipping it now have to be weighed against the long term interest that will need to be paid off over its lifetime, not to mention the debt itself.
+
+These decisions have to consider the business demands and the technical implications. They must be made by pod leads and developers in concert. By their nature it is impossible for either to do it on their own.
+
+It is the responsibility of developers to explain to leads the current state of technical debt and any implications it will have on current or future work. It is the responsibility of leads to explain to developers the priorities of the business and the urgency and importance of work.
+
+When it comes to paying off existing debt, focus on debt that currently has or is likely to have in the near future a "high rate of interest". Technical debt that is isolated can be safely left for another day.
+
+### 6. Little and often
+
+Technical debt should be paid of continuously in small payments. Exactly how much is up for debate. But here are some things to bare in mind:
+
+* Little and often over big and rarely.
+* Big payoffs may be required from time to time.
+* Avoid big refactors that drag on for more than one sprint, aka "Refactor Purgatory".
+* If a big rewrite is required do everything you can to transition to a new design incrementally.
+* Even if a sprint solely addresses tech debt it should be split into iterative chunks if possible.
+
+### 7. Aspire to be a boy scout
+
+> Always leave the campground cleaner than you found it.
```

Comment:
is it possible to make this stand out more? i see it as one of the most important things we should focus on